### DISCUSSION TITLE
Create a new Tab in the Account window with custom list of the Contacts related

### DISCUSSION THREAD

**Discussion by espe808** [YetiForceCompany/YetiForceCRM#14744] [How to]
I would to create in the Account window, a new tab with inside the list of the Contacts related to that Account, showing the contacts fields that I want.

How could I proceed?

![Schermata 2021-05-29 alle 16 57 58](https://user-images.githubusercontent.com/85027316/120074860-1c2d5c80-c09f-11eb-9348-03d49f79eaa6.png)

![Schermata 2021-05-29 alle 17 00 01](https://user-images.githubusercontent.com/85027316/120074938-8a721f00-c09f-11eb-9598-081748816ed0.png)

**Comment by vovpff** [comment]
You can change columns via admin panel - modules - relations

**Reply by espe808** [reply]
Thanks for the reply.
Ok for the columns, but how could I add a new tab like the existing tab?

![Schermata 2021-05-29 alle 16 57 58](https://user-images.githubusercontent.com/85027316/120086453-62a2ab80-c0df-11eb-9df5-084134ae5b92.png)

**Reply by vovpff** [reply]
You need to activate ability to add relations in /config files. I don't remember what is proper file name. 

At the Modules-Relations page blue button Add relation will appear.

What is your target?

**Reply by espe808** [reply]
I would to create a specific activity for the accounts (project 1, project 2, project 3...) with the list of related contacts inside these tabs/projects with specific columns for that projects.

Could you suggest me the right procedure?
Thanks

**Reply by vovpff** [reply]
Looks difficult. You need custom relation.
